src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -35,6 +38,40 @@ public void mercury_search_navigateToPageUsingSearchSuggestions() {
     Assertion.assertTrue(driver.getCurrentUrl().equals(suggestionLink));
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnMobile")
+  public void mercury_search_navigateUsingSearchSuggestionsOnMobile() {
+    String suggestionLink =
+        new ArticlePage()
+            .open(MercurySubpages.MAIN_PAGE)
+            .getTopBar()
+            .openSearch()
+            .typeInSearch(this.SEARCH_PHRASE)
+            .selectSearchSuggestion(0);
+
+    Assertion.assertTrue(driver.getCurrentUrl().equals(suggestionLink));
+    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MERCURY));
+  }
+
+  @Execute(onWikia = MercuryWikis.MEDIAWIKI_119)
+  @Test(groups = "mercury_search_navigateUsingSearchSuggestionsOnDesktop")
+  @InBrowser(browser = Browser.FIREFOX, browserSize = "1920x1080")
+  public void mercury_search_navigateUsingSearchSuggestionsOnDesktop() {
+    Search search =
+        new ArticlePage()
+            .open("/d/")
```

Comment:
@qaga - is there a const for it? I've seen some DIS tests, maybe this const is already upcoming to be ready to reuse?